tests/freqtradebot/test_freqtradebot.py

```diff
@@ -1265,14 +1265,14 @@ def test_exit_positions(mocker, default_conf_usdt, limit_order, is_short, caplog
     trades = [trade]
     freqtrade.wallets.update()
     n = freqtrade.exit_positions(trades)
-    assert n == 0
+    assert n == 1
```

Comment:
As mocked handle_trade is set to True, I don't get why we are expecting "n" (trades_closed) to be equal to 0. From what I see we should expect 1.

Reply:
With the current modifications, you're right.
`handle_trade()` was never called as `has_open_positions()` was true (the order is not filled) - hence it's return-value was really irrelevant.

I'll update this to return False which seems to be the better option for this particular test (which is both poorly setup, and is "a bit" odd.)